cli-plugins/apps/create-workload.hbs.md

```diff
@@ -7,22 +7,22 @@ This document describes how to create a workload from example source code with T
 The following prerequisites are required to use workloads with Tanzu Application Platform:
 
 - Install Kubernetes command line tool (kubectl). For information about installing kubectl, see [Install Tools](https://kubernetes.io/docs/tasks/tools/) in the Kubernetes documentation.
-- Install Tanzu Application Platform components on a Kubernetes cluster. See [Installing Tanzu Application Platform](../../install-intro.md).
-- Set your kubeconfig context to the prepared cluster `kubectl config use-context CONTEXT_NAME`.
-- Install Tanzu CLI. See [Install or update the Tanzu CLI and plug-ins](../../install-tanzu-cli.md#cli-and-plugin).
-- Install the apps plug-in. See the [Install Apps plug-in](install-apps-cli.md).
-- [Set up developer namespaces to use installed packages](../../set-up-namespaces.md).
+- Install Tanzu Application Platform components on a Kubernetes cluster. See [Installing Tanzu Application Platform](../../install-intro.hbs.md).
+- [Set your kubeconfig context](tutorials.hbs.md#changing-clusters) to the prepared cluster `kubectl config use-context CONTEXT_NAME`.
+- Install Tanzu CLI. See [Install or update the Tanzu CLI and plug-ins](../../install-tanzu-cli.hbs.md#cli-and-plugin).
+- Install the apps plug-in. See the [Install Apps plug-in](tutorials.hbs.md#install).
+- [Set up developer namespaces to use installed packages](../../set-up-namespaces.hbs.md).
 
```

Comment:
As you familiarize yourself with creating and managing the lifecycle of workloads on the platform, you may want to review the full Cartographer Workload spec to learn more about the fields/values that can be provided</br>
There are two methods for doing so:
1. On the [Cartographer docs website](https://cartographer.sh/docs/v0.6.0/reference/workload/) - detailed with comments
2. Via the terminal by running `kubectl explain workload.spec` - specific to the version running on the target cluster